### PR TITLE
Updates to custom settings and .gitignore

### PR DESCRIPTION
**Pull request by afk-mario**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,12 @@
...
+# Created by https://www.gitignore.io/api/vim
+
+### Vim ###
+[._]*.s[a-w][a-z]
+[._]s[a-w][a-z]
+*.un~
+Session.vim
+.netrwhist
+*~
+
```
Comment (rstacruz) [inline .gitignore:12]: I'd rather have editor-specific settings taken care of by a global ignore.

File: plugin/fastunite.vim [hunk 2]
```diff
@@ -32,6 +32,15 @@ if s:has_tag
...
 endif
 
+let g:unite_prompt = '  →  '
+
+"
+" Loaded Hook:
+"   allow the user to override some settings with a custom function
+"
+
+silent! call fastunite#loaded()
```
Comment (rstacruz) [inline plugin/fastunite.vim]: why bring this up?

Comment (afk-mario) [inline plugin/fastunite.vim]: I couldn't find another way to make the fastunite function call work and actually do something, I'm not sure if is the best approach.

Comment (rstacruz) [inline plugin/fastunite.vim]: actually i've been thinking of deprecating this. here's a better approach:

``` vim
runtime! plugin/fastunite.vim
" put customizations here
```

**Comment by rstacruz** [comment]
hehe, that's a lot of changes!

**Comment by afk-mario** [comment]
Oh sorry, no longer needed :)

**Comment by rstacruz** [comment]
Neat. Consider making your fork a full fledged project too, fix up the docs
and readme for new users  🎉😍

**Comment by afk-mario** [comment]
I will try to do that when denite has more features, thanks for your plugin 😄

**Pull request closed without merging**